std/string.d

```diff
@@ -3047,6 +3109,91 @@ if (isConvertibleToString!Range)
     cast(void) stripRight(ws.byUTF!wchar).array;
 }
 
+/++
+    Strips trailing characters as specified in second argument.
+
+    Params:
+        str = string or random access range of characters
+        chars = list of chars to strip
+
+    Returns:
+        slice of $(D str) stripped of trailing extra characters.
+
+    See_Also:
+        Generic stripping on ranges: $(REF _stripRight, std, algorithm, mutation)
+  +/
+auto stripRight(Range)(Range str, string chars)
+if (isSomeString!Range ||
+    isRandomAccessRange!Range && hasLength!Range && hasSlicing!Range &&
```

Comment:
Technically, we don't need to require a random access range here.  It is enough to require a bidirectional range and use `popBack` to strip from the end, in a similar way to `stripLeft`. In that case, we also don't need to require `hasLength` and `hasSlicing`, though of course in the implementation we can take advantage of those attributes where available.